apps/test/unit/templates/rubrics/RubricSettingsTest.jsx

```diff
@@ -170,8 +179,9 @@ describe('RubricSettings', () => {
       await act(async () => {
         await Promise.resolve();
       });
+      wrapper.update();
       expect(wrapper.find('Button').props().disabled).to.be.true;
-      expect(wrapper.text()).include(i18n.aiEvaluationStatus_pending());
+      expect(wrapper.text()).include(i18n.aiEvaluationStatus_in_progress());
```

Comment:
style nit: I realize you're just following an existing pattern here, but in the future, please use camelCase for js variable names where possible, e.g. `aiEvaluationStatusInProgress`. 

Most of our style rules are automatically enforced by prettier. the rest should be documented here: https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md

Reply:
No problem. Yeah, I was just matching the pattern from the pervious strings.